dialog/file.go

```diff
@@ -136,8 +137,11 @@ func (f *fileDialog) makeUI() fyne.CanvasObject {
 		}
 	})
 	buttons := widget.NewHBox(f.dismiss, f.open)
-	footer := fyne.NewContainerWithLayout(layout.NewBorderLayout(nil, nil, nil, buttons),
-		buttons, widget.NewHScrollContainer(f.fileName))
+	f.showHiddenCheck = widget.NewCheck("Show Hidden Files", func(changed bool) {
```

Comment:
I think this is what I saw in my comment in the conversation - this takes a lot of space away from the file panel above... Could it fit in the sidebar instead? (perhaps with the "Options" popout we suggested)